README.md

```diff
@@ -99,8 +99,9 @@ Use path settings to customize where USWDS Compile looks for USWDS source and ou
 | Setting                           | Default                                            | Description                                                                                                                                                                                     |
 | --------------------------------- | -------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `sprite.projectIconsOnly`         | `false`                                            | Include _only_ the icons in `paths.src.projectIcons` in the icon sprite.                                                                                                                        |
-| `settings.compile.browserslist`   | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | Define the [browserslist query](https://github.com/browserslist/browserslist?tab=readme-ov-file#queries) for CSS prefixes generated by [autoprefixer](https://github.com/postcss/autoprefixer). |
+| `settings.compile.browserslist`   | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | Define the [browserslist query](https://github.com/browserslist/browserslist?tab=readme-ov-file#queries) for CSS prefixes generated by [autoprefixer](https://github.com/postcss/autoprefixer).      |
 | `settings.compile.sassSourcemaps` | `true`                                             | Include sourcemap when compiling SASS to CSS.                                                                                                                                                   |
+| `settings.compile.quietSassDeps`  | `true`                                             | Silence USWDS Sass deprecation warnings. Your custom project Sass/SCSS will still trigger warnings if necessary.                                                                                |
```

Comment:
**suggestion:** Proposing a small tweak for clarity. 
```suggestion
| `settings.compile.quietSassDeps`  | `true`                                             | Silence USWDS Sass deprecation warnings. This will not silence deprecation warnings for project Sass.                                                                                |
```